app/models/container_image.rb

```diff
@@ -60,4 +60,12 @@ def perform_metadata_scan(ost)
   end
 
   alias_method :perform_metadata_sync, :sync_stashed_metadata
+
+  after_create :log_created_on
+
+  protected
+
+  def log_created_on
+    self.created_on ||= Time.now.utc
+  end
```

Comment:
1) So ```created_on``` might be a little misleading since the image was not created on this time but rather its the time  it 'discovered'. I think that either a comment or a rename of the field is needed.
2)  I like ```after_create```. its a much more elegant solution then putting this in the refresh parser but its not really our standard so im not sure if its ok. The other option is putting it here: https://github.com/zeari/manageiq/blob/fcce0b5fd68bbfe5c1cc6c924f29bd06abeef64f/app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb#L632-632